### PR TITLE
Implementing over commitment

### PR DESCRIPTION
**Pull request by a-belhadj**
- [x] Model
- [x] Forms
- [x] View (Table in ResourcePool + List Attribute from ResourcePool)
- [x] Test model

- [x] Test form
- [x] Documentation

### CODE REVIEW

**Comment by a-belhadj** [comment]
image
image
image

File: docs/manual/resource_tracking.md [hunk 1]
```diff
@@ -155,3 +155,13 @@ We link as well the pending instance(given by `squest.instance.id`) to this reso
...
+
+## Over commitment
+
+Over commitment is available on all resource pool attributes for consumers and producers.
+
+The over commitment allows you to specify whether resource pools produce/consume more or less than expected.
+
+The most common case is CPU/vCPU:
+
+If a host has two eight core processors and hyperthreading is enabled, that host will produces thirty-two vCPUs (8 cores x 2processors x 2 threads per core).
```
Comment (Sispheor) [inline docs/manual/resource_tracking.md:167]: then you can configure the over commitment on the CPU produced attribute to `2` to correspond to this behavior.

File: resource_tracker/forms.py [hunk 2]
```diff
@@ -116,6 +116,7 @@ class ResourcePoolForm(ModelForm):
 
     tags = TagField(label="Tags",
                     required=False,
+                    help_text="Comma-separated list of tags",
```
Comment (Sispheor) [inline resource_tracker/forms.py:119]: Add a section [after this one](https://hewlettpackard.github.io/squest/0.1/manual/resource_tracking/#link-service-catalog-instances-to-resources).
Explain what are tags.

And add a copy of this [tab]( https://django-taggit.readthedocs.io/en/latest/forms.html#tags-in-forms)

File: resource_tracker/views/resource_tracker_graph.py [hunk 3]
```diff
@@ -76,22 +76,22 @@ def create_resource_pool_svg(resource_pool: ResourcePool):
...
-                'display': attribute.get_total_produced(),
+                'display': round(attribute.get_total_produced(), 2),
                 'tooltip': f"Go to {attribute}'s producers",
                 'href': reverse(
                     'resource_tracker:resource_pool_attribute_producer_list',
                     kwargs={'resource_pool_id': resource_pool.id,
                             'attribute_id': attribute.id})},
             'consumed': {
-                'display': f"{attribute.get_total_consumed()}",
+                'display': round(attribute.get_total_consumed(), 2),
```
Comment (Sispheor) [inline resource_tracker/views/resource_tracker_graph.py:86]: round to integer

File: templates/generics/form_edit.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            {% endif %}
+        {% endfor %}
+    </div>
+{% endif %}
+<form class="form-horizontal" action="" method="post">{% csrf_token %}
+
+    {% for field in form %}
+        <div class="form-group {% if field.errors %}has-error{% endif %}">
+            <label for="{{ field.name }}" class="col-sm-2 control-label">
+                {{ field.label }}
```
Comment (Sispheor) [inline templates/generics/form_edit.html:15]: change help text life resource pool create template

**Review by Sispheor — CHANGES_REQUESTED** [review]
Some minor change to be done. Overall ok

**Comment by a-belhadj** [comment]
- [ ] Initial value at creation for over commitment
- [ ] Doc for taggit
- [ ] Round with no decimal
- [ ] Fix delete message on RG/RP (missing <ul> </ul> in RG)
- [ ] Change help text to label below input field
- [ ] Factorize RG/RP form for edit/create 
- [ ] Fix "for" attribute in label tag ( <label for="{{ field.id_for_label }}"...>) Click on label will bring you to the input

File: docs/manual/resource_tracking.md [hunk 5]
```diff
@@ -155,3 +155,47 @@ We link as well the pending instance(given by `squest.instance.id`) to this reso
...
+
+  * Otherwise, if there are any unquoted commas in the input, it will
+    be treated as comma-delimited. If not, it will be treated as
+    space-delimited.
+
+Examples:
+
+Tag input string       |   Resulting tags                |    Notes
+-----------------------|---------------------------------|-----------------------------------------------
+apple ball cat         |``["apple", "ball", "cat"]``     |No commas, so space delimited
```
Comment (Sispheor) [inline docs/manual/resource_tracking.md:184]: Space here after each pipe.  `| word`

**Comment by Sispheor** [comment]
Very good job done here. Thanks.
Code - Doc - Tests reviewed. All ok.
This is definitely an important feature for the v1.

**Review by Sispheor — APPROVED** [review]

**Pull request merged**